packages/mobile-web/src/tools/mobile-offline/offline-guidance/tool.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { z } from 'zod';
+import { Tool } from '../../tool.js';
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { ToolAnnotations } from '@modelcontextprotocol/sdk/types.js';
+import {
+  ExpertsReviewInstructionsSchema,
+  ExpertsReviewInstructionsType,
+  ExpertReviewInstructionsType,
+} from '../../../schemas/analysisSchema.js';
+import dedent from 'dedent';
+
+const EMPTY_INPUT_SCHEMA = z.object({}).describe('No input required');
+
+export class OfflineGuidanceTool implements Tool {
+  readonly name = 'Mobile Web Offline Guidance Tool';
+  readonly description =
+    'Delivers structured expert review instructions for detecting and remediating discrete offline violations in Lightning web components that require agentic analysis and contextual pattern recognition.';
```

Comment:
Nit: this is a bit nebulous and wandering for an MCP server tool description, given that MCP clients will effectively key on this description. My suggestion:

```suggestion
    'Provides structured review instructions to detect and remediate Mobile Offline code violations in Lightning web components (LWCs) for Salesforce Mobile Apps.';
```